src/backend/commands/alter.c

```diff
@@ -354,7 +355,7 @@ ExecRenameStmt(RenameStmt *stmt)
 		case OBJECT_MATVIEW:
 		case OBJECT_INDEX:
 		case OBJECT_FOREIGN_TABLE:
-			return RenameRelation(stmt);
+			return RenameRelation(pstate, stmt, isCompleteQuery);
```

Comment:
Please fix the indentation, the PG code convention is to use tab of length 4. See the other code in this file.